pages/index.njk

```diff
@@ -1,6 +1,7 @@
 ---
 title: Home
-description: Choose a category to test your web accessibility knowledge in. This quiz is brought to you by Sparkbox.
+seo_title: "Trivia11y: A Web Accessibility Quiz"
```

Comment:
I ran this by Kaitlyn for SEO, and she said we should have a mention of Sparkbox somewhere. Since the description is longer now, we can add "by Sparkbox" to this title, just for the home page. That way it'll show up in link previews and such.

Reply:
I've updated that in the latest commit.